samples/Samples.AspNetCoreMvc2/Samples.AspNetCoreMvc2.csproj

```diff
@@ -1,23 +1,12 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk.Web">
 
   <PropertyGroup>
-    <!-- override to remove net452 -->
-    <TargetFrameworks Condition="'$(OS)' == 'Windows_NT'">net461;netcoreapp2.1</TargetFrameworks>
-    <ApiVersion Condition="'$(ApiVersion)' == ''">2.0.4</ApiVersion>
-    <MvcRazorCompileOnPublish>true</MvcRazorCompileOnPublish>
-
-    <!-- Required to build multiple projects with the same Configuration|Platform, which is needed for the CI library/version matrix -->
-    <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
+    <TargetFrameworks>netcoreapp2.1</TargetFrameworks>
```

Comment:
Do you think we'll have a gap in testing real user scenarios by no longer testing Microsoft.AspNetcore 2.1 running on the .NET Framework?

Reply:
Actually, a VS "New Project" creates the same basic .csproj, so I think this change should be fine:

```
  <PropertyGroup>
    <TargetFramework>netcoreapp2.1</TargetFramework>
  </PropertyGroup>

  <ItemGroup>
    <PackageReference Include="Microsoft.AspNetCore.App" />
    <PackageReference Include="Microsoft.AspNetCore.Razor.Design" Version="2.1.2" PrivateAssets="All" />
  </ItemGroup>
```

Reply:
Unfortunately, we don't have telemetry for how many users run CoreMVC 2.x on .NET Framework, but using `Microsoft.AspNetCore.App` does require .NET Core.

Reply:
> Do you think we'll have a gap in testing real user scenarios?

I don't think so. The aspnet core libraries are the same on either run time (they are all built targeting `netstandard2.0`), and we know the CLR profiler works on both as well.